src/actions/index.js

```diff
@@ -15,35 +14,68 @@ export function receiveLocation(newLocation) {
   };
 }
 
+export function displaySuccess() {
+  return {
+    type: DISPLAY_SUCCESS,
+    message: 'Request succeeded',
+    color: 'green',
+  };
+}
+
+export function displayError(message) {
+  return {
+    type: DISPLAY_ERROR,
+    message,
+    color: 'red',
+  };
+}
+
+export function sendMessage() {
```

Comment:
this function name is pretty vague. we send a lot of messages, what kind of message is this?